src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -311,6 +326,8 @@ public function editAction($categoryId, Request $request)
             '@PrestaShop/Admin/Sell/Catalog/Categories/edit.html.twig',
             [
                 'categoryId' => $categoryId,
+                'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+                'enableSidebar' => true,
                 'allowMenuThumbnailsUpload' => $editableCategory->canContainMoreMenuThumbnails(),
```

Comment:
If the form type now handles the option `disable_menu_thumbnails_upload` is it still necessary to inject this in the template? Seems like it could be fetched from the form config directly no?